specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2132,6 +2135,28 @@
         }
       }
     },
+    "EncryptionIdentity":{
+       "description": "Identity to be used for server-side encryption on the storage account",
+       "properties": {
+         "userAssignedIdentity":{
+           "type": "string",
+           "description": "ResourceID of the UserAssignedIdentity to be associated with server-side encryption on the storage account.",
+           "x-ms-client-name": "UserAssignedIdentity"
+         },
+         "type": {
+          "type": "string",
+          "description": "The Encryption Identity type.",
+          "enum": [
+            "ManagedServiceIdentity",
+			      "StorageService"
+          ],
+          "x-ms-enum": {
+            "name": "EncryptionIdentityType",
+            "modelAsString": false
```

Comment:
Suggest to use "modelAsString": true, or it will be breaking change when add new value to the enum, and even will break before API version.